src/api/mappers/releaseMapper.ts

```diff
@@ -32,6 +33,20 @@ const releaseProducerFromDto = (dto: ReleaseProducerDto): ReleaseProducer => ({
   type: dto.type,
 });
 
+/**
+ * Maps minimum age to age rating.
+ * @param minAge Minimum age.
+ */
+const mapMinAgeToRating = (minAge: number | null): string => {
+  if (minAge !== null) {
+    if (minAge === 0) {
+      return 'All ages';
+    }
+    return `${minAge}+`;
+  }
+  return '';
```

Comment:
I think it's better better to avoid unnecessary nesting 😇
```suggestion
  if (minAge === null) {
    return '';
  }
  if (minAge === 0) {
    return 'All ages';
  }
  return `${minAge}+`;
```